tests/Benchmarks/Properties/AssemblyInfo.cs

```diff
@@ -19,3 +20,5 @@
 // COM, set the ComVisible attribute to true on that type.
 [assembly: ComVisible(false)]
 
+[assembly: MeasureInstructionsRetired]
+
```

Comment:
@jorive, I thought these results are captured by default now and hence we don't need to add this explicitly.